File: back/src/main/java/fr/uha/ensilan/concours/back/dto/message/PlayerJoinMessage.java
package fr.uha.ensilan.concours.back.dto.message;

import fr.uha.ensilan.concours.back.dto.user.UserDto;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class PlayerJoinMessage extends BaseMessage {
    private UserDto player;

    public PlayerJoinMessage() {
        super(MessageType.PLAYER_JOIN);
    }
}

File: front/src/app/core/domain/message/player-join.message.ts
import {BaseMessage} from "@domain/message/base.message";
import {MessageTypeEnum} from "@domain/message/message-type.enum";
import {UserDto} from "@domain/user.dto";

export class PlayerJoinMessage extends BaseMessage {
  override readonly type = MessageTypeEnum.PLAYER_JOIN;
  player: UserDto;
}
